volume/drivertest/drivertest.go

```diff
@@ -241,10 +241,20 @@ func DriverTestSnapshots(c *C, drivername volume.DriverType, root string, args [
 	// Write another file
 	writeExtra(c, driver, vol)
 
+	// Get the tenant volume of the snapshot that doesn't exist
+	tvol, err := driver.GetTenant("Base_Snap2")
+	c.Assert(err, Equals, volume.ErrVolumeNotExists)
+	c.Assert(tvol, IsNil)
+
 	// Re-snapshot with the extra file
 	err = vol.Snapshot("Snap2", "snapshot-message-1", []string{"tag1", "tag2", "tag3"})
 	c.Assert(err, IsNil)
 
+	// Get the tenant volume of the snapshot
+	tvol, err = driver.GetTenant("Base_Snap2")
+	c.Assert(err, IsNil)
+	c.Assert(tvol.Name(), Equals, "Base")
+
```

Comment:
The description on the interface says:
```
GetTenant returns the parent volume or the volume if it is the parent
```
So that sounds like there are 2 happy cases: (1) `GetTenant(nonParentVolumeName)` and (2) `GetTenant(parentVolumeName)`, but looks lik we're only testing one.